test/unit/groupTest.js

```diff
@@ -301,22 +325,22 @@ test('randomizing property of a group sets to individually random properties', a
   nativeAPI.setAnimationSpriteSheet("CAT", 0, {}, () => {});
   nativeAPI.makeNewDanceSpriteGroup(5, 'CAT', 'circle');
 
-  nativeAPI.layoutSprites('CAT', 'grid');
   let cats = nativeAPI.getGroupByName_('CAT');
   for (let i = 0; i < cats.length; i++) {
     nativeAPI.setProp(cats[i].name, 'scale', 20);
   }
 
   nativeAPI.setPropRandomEach(cats, 'scale');
 
-  let catScales = [];
+  // Create a list of cat scales and fail if at least one is not unique
+  // This test will fail when all five scales are randomly changed to
+  // the same value and should be re-run in those flakey cases.
```

Comment:
Can you add to this comment a note that we expect failures to be extremely rare? That way if for some reason we start seeing this test be more flaky than expected we can investigate.